Reconstruction/RecInterface/RecInterface/ITowerTool.h

```diff
@@ -0,0 +1,67 @@
+#ifndef RECINTERFACE_ITOWERTOOL_H
+#define RECINTERFACE_ITOWERTOOL_H
+
+// Gaudi
+#include "GaudiKernel/IAlgTool.h"
+
+// datamodel
+#include "datamodel/CaloCluster.h"
+
+/** @class ITowerTool RecInterface/RecInterface/ITowerTool.h ITowerTool.h
+ *
+ *  Abstract interface to tower building tool.
+ *
+ *  @author Anna Zaborowska
+ */
+
+class ITowerTool: virtual public IAlgTool {
+public:
+  DeclareInterfaceID(ITowerTool, 1, 0);
+  /**  Find number of calorimeter towers.
+   *   @return Array containing number of towers in eta and phi.
+   */
+  virtual std::array<int,2> towersNumber() = 0;
```

Comment:
I'd prefer to use a struct that explicitly has the members `nEta`, `nPhi` (since some of the rest of the tool interface is specific)